### PR TITLE
New blackbird component

### PR DESCRIPTION
**Pull request by koolsb**
**Description:**

Documentation for new Monoprice Blackbird component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/media_player.blackbird.markdown [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    sources:
+      1: 
+        name: BluRay
+      3:
+        name: Chromecast
+      8:
+        name: AppleTV
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/media_player.blackbird.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/media_player.blackbird.markdown [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+title: "Blackbird 8x8 HDMI Matrix Switch"
+description: "Instructions on how to integrate Monoprice Blackbird 4k 8x8 HDBaseT Matrix Switch into Home Assistant."
+date: 2018-03-23 16:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: monoprice.svg
+ha_category: Media Player
+ha_release: 0.66
```
Comment (frenck) [inline source/_components/media_player.blackbird.markdown]: This won't make it into 0.66, maybe set it to 0.67 already.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
@koolsb: Thanks for the PR!
Looks good in general, nevertheless, left some comments.
Could you please take a look? 👍

**Comment by koolsb** [comment]
Thanks for the feedback @frenck! Let me know how this looks!

**Pull request closed without merging**